### DISCUSSION TITLE
Is anybody working on adding support for adding stream from external sources?

### DISCUSSION THREAD

**Discussion by jsalgado10** [jellyfin/jellyfin#8290] [Q&A] [answered]
I am wondering if anybody is working on adding support to stream media directly from hosters and debrid sources. I have used Kodi for years, but the downside of it is that the application needs to be installed on the device and configure as well. Whereas with Jellyfin, you can use the web client or apps with minimum setup. Currently, the media file has to reside on the server, and you are limited to the hard drive space. 

Now, it would be nice to have the ability to play media from other sources without having to store the media file on the server

I am interested in making a plugin that will accomplish this and will be happy to work together with the jellyfin team.

**Comment by cvium** [comment, marked as answer]
We will not be supporting anything more complicated than .strm files

**Reply by jsalgado10** [reply]
That works for me. will work on a prototype and provide an update

**Reply by cvium** [reply]
Just to clarify: We probably won't be supporting any remote playback that can't be handled with .strm files or ffmpeg

**Reply by jsalgado10** [reply]
Yes, i understand. Using strm files is what i plan on doing. will try to filter out only media supported

**Reply by jsalgado10** [reply]
Are any of the clients supporting extra info when viewing details on the media? Since there will be multiple resources available where the media can be streamed, I want to give the user the option to select which one they would like to use.

Thanks
